dev/com.ibm.ws.microprofile.reactive.messaging.kafka/resources/com/ibm/ws/microprofile/reactive/messaging/kafka/resources/ReactiveMessaging.nlsprops

```diff
@@ -55,6 +55,11 @@ kafka.groupid.not.set.CWMRX1005E=CWMRX1005E: The group.id property was not set i
 kafka.groupid.not.set.CWMRX1005E.explanation=The Kafka consumer group ID must be set to allow Reactive Messaging to function properly.
 kafka.groupid.not.set.CWMRX1005E.useraction=Set the property in one of the MicroProfile ConfigSource locations for this application, for example, the microprofile-config.properties file.
 
+#Do not translate "kafka-clients", "Apache Kafka" or "Maven Central"
+kafka.library.not.present.CWMRX1006E=CWMRX1006E: Classes from the kafka-clients jar could not be loaded. Ensure that the kafka-clients jar and its dependencies are available as libraries to your application.
```

Comment:
Change to
"Classes from the kafka-clients JAR file could not be loaded. Ensure that the kafka-clients JAR file and its dependencies are available as libraries to your application."